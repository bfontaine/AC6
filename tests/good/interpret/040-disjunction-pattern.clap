val classify = case { A  or B => 1 | C => 2 }
val xA = classify (A)
val xB = classify (B)
val xC = classify (C)

