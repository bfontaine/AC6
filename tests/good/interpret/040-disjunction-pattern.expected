:- classify = <code>
:- xA = 1
:- xB = 1
:- xC = 2
